GenderPayGap.WebUI/Views/ActionHubOld/MixedResult.cshtml

```diff
@@ -0,0 +1,142 @@
+            
+                <h4 class="govuk-heading-s">@(referenceIndex++). Diversity training</h4>
+                <ul class="govuk-list">
+                    <li>
+                        Bezrukova, K., Spell, C.S., Perry, J., & Jehn, K. (2016). A meta-analytical integration of over 40 years of research on diversity training evaluation. Psychological Bulletin, 142(11), 1227–1274.
+                    </li>
+                    <li>
+                        Dobbin, F., & Kalev, A. (2016). Why diversity programs fail. Harvard Business Review, 94(7/8), 52-60.
+                    </li>
+                </ul>
+            
+                <h4 class="govuk-heading-s">@(referenceIndex++). Leadership development training</h4>
+                <p class="govuk-body">Not available.</p>
+            
+                <h4 class="govuk-heading-s">@(referenceIndex++). Performance self-assessments</h4>
+                <ul class="govuk-list">
+                    <li>
+                        Fletcher (1999). The implications of research on gender differences in self-assessment and 360 degree appraisal. Human Resource Management Journal, 9(1), 39-46.
+                    </li>
+                    <li>
+                        Beyer (1990). Gender differences in accuracy of self-evaluations of performance. Journal of Personality and Social Psychology, 59(5), 960-970.
+                    </li>
+                </ul>
+            
+                <h4 class="govuk-heading-s">@(referenceIndex++). Diverse selection panels</h4>
+                <ul class="govuk-list">
+                    <li>
+                        De Paola, M., & Scoppa, V. (2015). Gender discrimination and evaluators’ gender: evidence from Italian academia. Economica, 82(325), 162-188.
+                    </li>
+                    <li>
+                        Bagues, M., Sylos-Labini, M., & Zinovyeva, N. (2017). Does the gender composition of scientific committees matter? American Economic Review, 107(4), 1207-38.
+                    </li>
+                    <li>
+                        Abramo, G., D’Angelo, C. A., & Rosati, F. (2015). Selection committees for academic recruitment: does gender matter? Research Evaluation, 24(4), 392-404.
+                    </li>
+                    <li>
+                        Duguid, M. (2011). Female tokens in high-prestige work groups: Catalysts or inhibitors of group diversification? Organizational Behavior and Human Decision Processes, 116(1), 104-115.
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </div>
+</div>
+
+@{
+    var leftLink = new PaginationLinkViewModel
+    {
+        Title = "Previous",
+        Text = "Promising actions",
+        Url = Url.Action("Promising", "ActionHub")
+    };
+
+    await Html.RenderPartialAsync("/Views/Components/PaginationLinks/PaginationLinks.cshtml", new PaginationLinksViewModel
+    {
+        LeftLink = leftLink,
+        RightLink = null

```

Comment:
it really bugs me how this drops you off a cliff and doesn't give you any logical onward journeys... again not one for this PR though! :D

Reply:
Agreed! It's a bit weird, but I think the chunkier redesign work will happen at some point soon-ish and that's when stuff like this will be tackled! 😄